doc/en/conf.py

```diff
@@ -23,9 +23,11 @@
 from textwrap import dedent
 from typing import TYPE_CHECKING
 
-from _pytest import __version__ as version
+from _pytest import __version__ as full_version
 
 
+version = full_version.split("+")[0]
```

Comment:
@RonnyPfannschmidt this results in the `|release|` RST substitution having the value of `8.2` which might have a weird perception in changelog draft titles: https://docs.pytest.org/en/8.2.x/changelog.html#to-be-included-in-vrelease-if-present / https://docs.pytest.org/en/latest/changelog.html#to-be-included-in-vrelease-if-present.

Should this be something like `+dev` in the end but without a variable component, something static? Though, perhaps in RTD it'd be okay to use the long original version.

Reply:
Good point, I cut off too much

Reply:
Good point, I cut off too much